scanners/kmp/kmp.go

```diff
@@ -90,6 +91,10 @@ func (s *Scanner) DetectPlatform(searchDir string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	if gradleProject == nil {
```

Comment:
This is a fix for the current KMP scanner, `gradle.ScanProject` can return nil error and nil gradleProject, if the necessary project files weren't found.